src/SFA.DAS.ApplyService.Data/Queries/OversightReviewQueries.cs

```diff
@@ -237,7 +237,7 @@ public async Task<ApplicationOversightDetails> GetOversightApplicationDetails(Gu
                               FROM Apply apply
 	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
                           LEFT JOIN OversightReview outcome ON outcome.ApplicationId = apply.ApplicationId
-						  LEFT OUTER JOIN contacts on contacts.ApplyOrganisationId = org.Id
+						  LEFT OUTER JOIN contacts on contacts.ApplyOrganisationId = org.Id and apply.CreatedBy = contacts.id
```

Comment:
I'm not sure what this is doing but looks all part of the process

Reply:
ah that's a left over from the different email> application stuff, I'm taking it out